esp/src/src-react/components/FileParts.tsx

```diff
@@ -7,7 +7,11 @@ import { useFluentGrid } from "../hooks/grid";
 import { useFile } from "../hooks/file";
 import { HolyGrail } from "../layouts/HolyGrail";
 
-const formatNum = d3Format(",");
+const formatNum = (num) => {
+    const format = d3Format(",");
+    if (num < 0) return nlsHPCC.NotAvailable;
+    return format(num);
+};
```

Comment:
Same as above - but given this is code duplication, I would now suggest moving this into a util.js or common file?